docs/lifecyclehook.md

```diff
@@ -60,3 +60,5 @@ spec:
      http:
        url: http://dummy.restapiexample.com/api/v1/employees
 ```
+
+> You could provide arbitrary numbers of hooks. The exit is a special hook, for others the expression is mandatory.
```

Comment:
What are you trying to explain or clarify by adding these two sentences to the end of the doc? I'm not sure what additional information this provides.

Reply:
I don't quite understand the meaning of `exit` lifecycle hook. And don't understand what else hooks I can use. After reading the code, I finally know the `exit` hook is a special one. And for others, the expression is necessary. It will panic if the expression is empty.

Please correct me if I misunderstand it.

Reply:
I think I see what you mean. 

I would delete this line 64 that you added and instead insert the lines above that I suggest.

Reply:
Ah, I can't comment on the lines above since they weren't part of the PR, so will add here:

Add this on line 9 or 10:
> Put differently, an exit handler is like a workflow-level `LifecycleHook` with an expression of `workflow.status == "Completed"` or `workflow.status == "Failed"` or `workflow.status == "Error"`.

Reply:
@LinuxSuRen - can you please delete the line 64 that you added, and instead add the above to your PR when you have a moment and we'll approve for merge. Thanks!

Reply:
hi @caelan-io , just a friendly ping. Please feel free to check it once you have free time.